docs/resources/workspace_desktop.md

```diff
@@ -0,0 +1,191 @@
+
+* `data_volume` - (Optional, List) Specifies the configuration of data volumes.
+  The [object](#desktop_volume) structure is documented below.
+
+* `availability_zone` - (Optional, String, ForceNew) Specifies the availability zone where the desktop is located.
+  Changing this will create a new resource.
+
+* `security_groups` - (Optional, String, ForceNew) Specifies the ID list of security groups.
+  In addition to the custom security group, it must also contain a security group called **WorkspaceUserSecurityGroup**.
+  Changing this will create a new resource.
+
+* `user_group` - (Required), String, ForceNew) Specifies the user group to which the desktop belongs.
+  The valid values are as follows:
+  + **sudo**: Linux administrator group.
+  + **default**: Linux default user group.
+  + **administrators**: Windows administrator group.
+  + **users**: Windows standard user group.
+
+  Changing this will create a new resource.
+
+* `nic` - (Optional, List, ForceNew) Specifies the NIC information corresponding to the desktop.
+  The [object](#desktop_nic) structure is documented below. Changing this will create a new resource.
+
+* `name` - (Optional, String, ForceNew) Specifies the desktop name.
+  The name can contain `1` to `15` characters, only letters, digits and hyphens (-) are allowed.
+  The name must start with a letter or digit and cannot end with a hyphen.
+  Changing this will create a new resource.
+
+* `name_prefix` - (Optional, String, ForceNew) Specifies the name prefix. Changing this will create a new resource.
+
+* `email_notification` - (Optional, Bool, ForceNew) Specifies whether to send emails to user mailbox during important
+  operations. Changing this will create a new resource.
+
+* `tags` - (Optional, Map, ForceNew) Specifies the key/value pairs of the desktop.
+  Changing this will create a new resource.
+
+* `delete_user` - (Optional, Bool) Specifies whether to delete user associated with this desktop after deleting it.
+  The user can only be successfully deleted if the user has no other desktops.
+  Changing this will create a new resource.
+
+<a name="desktop_volume"></a>
+The `root_volume` and `data_volume` block supports:
+
+* `type` - (Required, String) Specifies the type of system volume.
+  The valid values are as follows:
+  + **SAS**: High I/O disk type.
+  + **SSD**: Ultra-high I/O disk type.
+
+  -> Updates are not supported for this parameter. Changing this will not create a new resource, but will throw an
+     error.
+
+* `size` - (Required, Int) Specifies the size of system volume, in GB.
+  + For root volume, the valid value is range from `80` to `1,020`.
+  + For data volume, the valid value is range from `10` to `8,200`.
+
+<a name="desktop_nic"></a>
+The `nic` block supports:
+
+* `network_id` - (Required, String, ForceNew) Specifies the network ID.
+  The valid values are as follows:
+  + **SAS**: High I/O disk type.
+  + **SSD**: Ultra-high I/O disk type.
```

Comment:
the values are not correct, and it's not forcenew in the schema

Reply:
fixed.